22_pipeslab/pipeslab.txt

```diff
@@ -1,34 +1,78 @@
 For each of the following, type a single command line that performs:
 
-  1. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner.
+ 
+ 1. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner.
   Answer: 
 
-  2. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner, group and others.
+ 
+
+ls -l | cut -d- -f1 |grep d |wc -l 

```

Comment:
This prints only directories. You can do better with `cut -c` which prints specific characters, so this one is more precise: `ls -l | cut -c4 | grep -c x